src/Sulu/Component/PHPCR/Tests/Unit/PathCleanupTest.php

```diff
@@ -44,15 +45,34 @@ protected function setUp(): void
                 'en' => [
                     '&' => 'and',
                 ],
-            ]
+                'bg' => [
+                    '&' => 'и',
+                ],
+            ],
+            new AsciiSlugger()
         );
     }
 
-    public function testCleanup()
+    /**
+     * @dataProvider cleanupProvider
+     */
+    public function testCleanup($a, $b, $locale)
     {
-        $clean = $this->cleaner->cleanup('-/aSDf     asdf/äöü-/hello: world\'s', 'de');
+        $clean = $this->cleaner->cleanup($a, $locale);
+        $this->assertEquals($b, $clean);
+    }
 
-        $this->assertEquals('/asdf-asdf/aeoeue/hello-worlds', $clean);
+    public function cleanupProvider()
```

Comment:
can we add a testcase for `multiple dashes`, `dash before slash`, `dash after slash`, `dash at the beginning, `dash at the end` and `multiple slashes`? 😇

Reply:
Will add test for theses